.github/workflows/build-and-test.yml

```diff
@@ -138,8 +138,8 @@ jobs:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
         with:
           status: failure
-          channel: "#terra-tsps-alerts"
-          username: "TSPS push to main branch"
+          channel: "#terra-teaspoons-alerts"
```

Comment:
how hard is it to change slack channel names?

Reply:
when should we make this change? right before merging this PR?